src/api/topics/topic.entity.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('topics')
+export class TopicEntity {
+  @PrimaryGeneratedColumn()
```

Comment:
WDYT about creating like a "base" entity with common attributes? Something like id, create at, updated at. And then you can just extend it with all other entities.